src/containers/change-requests-table/change-requests-table.tsx

```diff
@@ -3,58 +3,27 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
-import { useEffect, useState } from 'react';
-import { AxiosResponse } from 'axios';
-import { ChangeRequest } from 'utils';
-import { apiFetch } from '../../shared/axios';
-import { booleanPipe, fullNamePipe, wbsPipe } from '../../shared/pipes';
 import CRTable from '../../components/change-requests-table/change-requests-table'; // Directly rename the default import
-import { DisplayChangeRequest } from '../../components/change-requests-table/change-requests-table';
 import './change-requests-table.module.css';
+import { useAllChangeRequests } from '../../services/change-requests';
 
 const ChangeRequestsTable: React.FC = () => {
-  const [allChangeRequests, setAllChangeRequests] = useState<DisplayChangeRequest[]>([]); // store projects data
 
-  // Transforms given change request data and sets local state
-  const updateData: (response: AxiosResponse) => void = (res) => {
-    setAllChangeRequests(
-      res.data.map((cr: ChangeRequest) => {
-        return {
-          id: cr.id,
-          submitterName: fullNamePipe(cr.submitter),
-          wbsNum: wbsPipe(cr.wbsNum),
-          type: cr.type,
-          dateReviewed: cr.dateReviewed ? new Date(cr.dateReviewed).toLocaleDateString() : '',
-          accepted: cr.accepted ? booleanPipe(cr.accepted) : '',
-          dateImplemented: cr.dateImplemented
-            ? new Date(cr.dateImplemented).toLocaleDateString()
-            : ''
-        };
-      })
```

Comment:
I believe this is the code that you'll need to use to transform a `ChangeRequest` array into a `DisplayChangeRequest` array.